src/test/java/com/aws/greengrass/localdebugconsole/DashboardServerTest.java

```diff
@@ -5,12 +5,12 @@
 
 package com.aws.greengrass.localdebugconsole;
```

Comment:
I couldn't tell - are there any tests that check the correct ports are read from the config?

Reply:
No, there aren't any specific tests for that at this time.